cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -499,3 +499,454 @@ def test_create_snapshot_via_ae(appliance, request, domain, small_test_vm):
+    Bugzilla:
+        1429313
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_sui_create_snapshot_from_vm_details_page_snapshot_page_and_service_details_page():
+    """
+    Snapshot can be created from VM details page, service details page
+    and snapshot page.
+    Check all pages and the snapshot count displayed on vm details page.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: SelfServiceUI
+        caseimportance: medium
+        initialEstimate: 1/4h
+        startsin: 5.9
+        title: SUI: Create snapshot from vm details page, snapshot page
+               and service details page
+        setup:
+            1. Add a vmware type provider
+            2. Create and order a service catalog which provisions a VM
+            3. Login to SSUI
+        testSteps:
+            1. Test that snapshot can be created from Service Details page
+                * Navigate to My Services -> <service_name>
+                * Click Snapshots -> Create
+                * Fill Name and click Create
+            2. Test that snapshot can be created from VM Details page
+                * Navigate to My Services -> <service_name> -> <vm_name>
+                * Click Snapshots -> Create
+                * Fill Name and click Create
+            3. Test that snapshot can be created from Snapshot page
+                * Navigate to My Services -> <service_name> -> <vm_name> -> Snapshots
+                * Click Configuration -> Create Snapshot
+                * Fill Name and click Create
+        expectedResults:
+            1. Snapshot successfully created
+            2. Snapshot successfully created
+            3. Snapshot successfully created
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_notification_for_snapshot_delete_failure_ec2():
+    """
+    Requires ec2 access via web-ui.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Provisioning
+        caseimportance: medium
+        initialEstimate: 1/4h
+        title: test notification for snapshot delete failure
+        setup:
+            1. Add ec2 provider
+        testSteps:
+            1. Create a snapshot on EC2 provider
+            2. Try to delete snapshot via CFME UI
+        expectedResults:
+            1. Snapshot created
+            2. Snapshot not deleted and notification displayed
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_snapshot_timeline_group_actions():
+    """
+    Test the SUI snapshot timeline.
+    Test grouping of actions in a timeline. Try to create a couple of
+    snapshots in a rapid succession, check how it looks in the timeline.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/3h
+        setup:
+            1. Add vmware provider
```

Comment:
Test function should be marked only for vmware if that's the relevant provider for the test.

Reply:
Added markers; changed required providers.